app/controllers/activejob_web/jobs_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ActivejobWeb
+  class JobsController < ApplicationController
+    before_action :set_job, only: %i[show edit update]
+
+    def index
+      @jobs = ActivejobWeb::Job.all
+    end
+
+    def show; end
+
+    def edit; end
+
+    def update
+      @job.approver_ids = params.dig(:activejob_web_job, :approver_ids).compact_blank.map(&:to_i)
+      @job.executor_ids = params.dig(:activejob_web_job, :executor_ids).compact_blank.map(&:to_i)
```

Comment:
@GowthamK01 Please define a method named `job_params` and permit these attribute and use this method for mass assignment